packages/flutter_tools/test/integration.shard/test_data/multidex_project.dart

```diff
@@ -179,7 +179,7 @@ class MultidexProject extends Project {
       defaultConfig {
           // TODO: Specify your own unique Application ID (https://developer.android.com/studio/build/application-id.html).
           applicationId "com.example.multidextest2"
-          minSdkVersion 16
```

Comment:
cc @GaryQian 

Instead of changing the `minSdkVersion`, what about changing the constraint of the plugins? 

```
cloud_firestore: 2.5.3
firebase_core: 1.6.0
```

Reply:
Pinning to known good dependency versions sounds like the better solution to me.